core/src/main/java/kafka/server/share/DelayedShareFetch.java

```diff
@@ -55,6 +60,7 @@ public class DelayedShareFetch extends DelayedOperation {
     private final ReplicaManager replicaManager;
 
     private Map<TopicIdPartition, FetchRequest.PartitionData> topicPartitionDataFromTryComplete;
+    private Map<TopicIdPartition, FetchPartitionOffsetData> logReadResponseFromTryComplete;
```

Comment:
Do we need the variable name to have suffix `TryComplete`? I don't find the suffix is any helpful.